### PR TITLE
PoS: fix tendermint inactive validator set update - 0.14.0

### PR DESCRIPTION
**Pull request by tzemanovic**
forward port of #1144 for 0.14.0

### CODE REVIEW

File: proof_of_stake/src/tests.rs [hunk 1]
```diff
@@ -755,27 +758,61 @@ fn test_validator_sets() {
...
+            s,
+            &params,
+            addr,
+            stake,
+            epoch,
+            params.pipeline_len,
+        )
+        .unwrap();
+
+        update_validator_deltas(s, &params, addr, stake.change(), epoch)
```
Comment (brentstone) [inline proof_of_stake/src/tests.rs:790]: is it useful at all (for consistency) to update total deltas too? Maybe not necessary for the test.

**Review by brentstone — APPROVED** [review]
LGTM!

**Comment by Fraccaman** [comment]
pls spawn devnet [anoma-devnet-draft,3,heliaxdev@1be9883,ON]

**Comment by Fraccaman** [comment]
pls spawn devnet [anoma-devnet-draft,3,heliaxdev@1be9883,ON]

**Comment by Fraccaman** [comment]
pls spawn devnet [anoma-devnet-draft,3,heliaxdev@1be9883,ON]

**Pull request merged**